api/cas-server-core-api-authentication/src/main/java/org/apereo/cas/authentication/MultiInstanceMfaAuthenticationHandler.java

```diff
@@ -0,0 +1,30 @@
+package org.apereo.cas.authentication;
+
+/**
+ * Interface implemented by AuthenticationHandlers that are used to authenticate credentials
+ * that are created by MFA providers that can configure multiple instances in the CAS server.
+ * The id of the MFA provider instance is used to determine if the instance of the
+ * AuthenticationHandler can authenticate the proposed credential.
+ *
+ * @author Travis schmidt
+ * @since  5.3.4
```

Comment:
I guess you meant 6.0.0...

Reply:
This is included in PR #3498 in 5.3.x, so it should be part of 5.3.4 release

Reply:
Indeed